packages/leancode_debug_page/README.md

```diff
@@ -1,6 +1,25 @@
 # leancode_debug_page
 
-A debug page that logs http requests and logger logs.
+A debug page that gathers http requests and logger logs. Features:
+
+- Detailed information about requests (request, response) and logs
+- Filtering requests by
+  - Status code
+  - Search
+- Filtering logs by
+  - Log level
+  - Search
+- Sharing
+  - All logs / requests
+  - Individual items
+- Two configurable entry points
+  - Draggable floating action button
+  - Device shake
+
+![Inspector](images/requests.png "Inspector")
+![Request details](images/request_details.png "Request details")
+![Logs](images/logs.png "Logs")
```

Comment:
Maybe these images could be moved to a specific item in `Features`? Or at least some captions would be nice

Reply:
captions added